en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/gateway-policies.md

```diff
@@ -0,0 +1,98 @@
+
+    - If you click on the recently attached **AH (Add Header)** policy, you'll still have access to view and edit the initially entered values.
+
+        [![View added common policy]({{base_path}}/assets/img/deploy/gateway/global-polices-add-header-policy-view.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-add-header-policy-view.png)
+
+    - Let's proceed by adding several more policies to the existing Request Flow. Choose any number of policies from the Request tab within the Policy List.
+
+        [![Add multiple common policies to a flow]({{base_path}}/assets/img/deploy/gateway/global-polices-add-multiple-policies.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-add-multiple-policies.png)
+
+        !!! note
+            - You can rearrange the dropped policies that are attached to the Request Flow.
+            - You can download the policy source as a .zip file by clicking the cloud download icon.
+            - Clicking on the delete icon cancels the dropped policy.
+
+4. When you have finalized the dragged and dropped policies to your preference, proceed by clicking the **Save** button situated at the bottom of the page.
+
+    [![Save global policy]({{base_path}}/assets/img/deploy/gateway/global-polices-save-policy-mapping.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-save-policy-mapping.png)
+
+## Step 2 - Deploy a gateway policy
+
+- Upon the existence of at least one gateway policy in the queue, it will be displayed in the **Global Policies** tab.
+
+    [![Global policy listing]({{base_path}}/assets/img/deploy/gateway/global-polices-list.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-list.png)
+
+- Click on the expandable button at the beginning of the policy entry row to choose from available gateway labels for deploying the respective global policy.
+
+    !!! note
+        The selection will not display gateway labels that have active global policy deployment. This is because a particular gateway can have only one active gateway policy deployment.
+
+- After selecting the required gateway labels, click the **Deploy** button. A confirmation dialog box will appear, and upon clicking the **Deploy** button within the dialog, the selected gateway policy will be deployed to the chosen gateway environments.
+
+    [![Deploy global policy]({{base_path}}/assets/img/deploy/gateway/global-polices-deploy-to-gateway.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-deploy-to-gateway.png)
+
+To undeploy a gateway policy from any gateway environment, simply click on the cloud off button placed in front of each gateway label. Similarly, just as during deployment, an undeployment confirmation dialog box will prompt you for confirmation.
```

Comment:
```suggestion
To undeploy a global policy from any gateway environment, simply click on the cloud off button placed in front of each gateway label. Similarly, just as during deployment, an undeployment confirmation dialog box will prompt you for confirmation.
```

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7526/commits/64bafa0aa418ed0dbbc05491694766f9d24f000b.
Thank you for your suggestion.